api/views.py

```diff
@@ -70,21 +69,21 @@ def consume_file(request):
                                 else:
                                         word_frequencies[word.text] += 1
                 
-#print(word_frequencies)
+        #print(word_frequencies)
```

Comment:
Remove commented out code

Reply:
done

Reply:
Resolve conflicts with the main branch and I will merge it